pages/howto/easybuild.md

```diff
@@ -0,0 +1,374 @@
+# An Easybuild Example  
+
+We recommend CCR users utilize the Easybuild framework when installing software.  This tool ensures that the compilation and linking that happens during a software installation is done correctly and will work with our systems.  We do not offer instruction on using Easybuild.  However, there is excellent [documentation](https://docs.easybuild.io/) and [tutorials](https://tutorial.easybuild.io/) provided by Easybuild developers.  
+
+## Easybuild Background  
+
+At CCR, we provide a set of [supported toolchains](../software/releases.md) and users are able to use Easybuild to install software for themselves without administrative privileges.  We provide instructions to get started [here](../software/building.md).  Some packages install quite easily and the `samtools` example provided in our documentation is one of those packages.  However, what happens if you want to install something that CCR doesn't offer, but you're not finding an exact match for a toolchain provided by CCR or the version of the software you want to run?  It's a bit of trial and error, to be honest.  And this is what CCR staff are doing on a daily basis as we work through the [installation requests](../software/building.md#software-build-requests) in our [GitHub repository](https://github.com/ubccr/software-layer/issues).
+
+In this how-to section, we're going to take you through the process to provide you with the information you'll need to be off and running on your own Easybuild installation!
+
+!!! Danger "STOP!"
+    Before going forward, ensure that you've read the documentation on using the [new modules](../software/modules.md).  Are you SURE this software isn't already installed in the latest software release? Use the `module spider` command to search.  You can also check the CCR [GitHub repo](https://github.com/ubccr/software-layer/issues) for open & closed build requests.  Read the information about [building your own software](../software/building.md) and setting up a build environment.  If you're planning to make this software accessible to your entire research group, ensure you've got [this setup](../software/building.md#building-modules-for-your-group) correctly.  
```

Comment:
Suggest using alternate verbiage here. Instead of "the new modules", let's use "the latest ccrsoft release".

Reply:
Updated